packages/components-native/src/Divider/Divider.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { View } from "react-native";
+import { styles } from "./Divider.style";
+
+interface DividerProps {
+  /**
+   * The weight of the divider.
+   *
+   * @default "base"
+   */
+  readonly size?: "base" | "large" | "largest";
```

Comment:
I wonder if

```suggestion
  readonly size?: keyof typeof styles;
```

would work here 🤔

Reply:
That would key rid of `base` and add `divider`. We may be able to do something like
```
export const styles = StyleSheet.create({
  divider: {
    height: JobberStyle["space-minuscule"],
    margin: 0,
    borderBottomWidth: JobberStyle["border-base"],
    borderBottomColor: JobberStyle["color-border"],
  },
  sizes: {
    large: {
      borderBottomWidth: JobberStyle["border-thick"],
      opacity: 0.875,
    },
    largest: {
      borderBottomWidth: JobberStyle["space-small"],
      opacity: 0.375,
    },
  }
});
```

```
readonly size?: keyof typeof styles.sizes;
```
Though

Reply:
This would work though

```diff
diff --git a/packages/components-native/src/Divider/Divider.style.ts b/packages/components-native/src/Divider/Divider.style.ts
index 72c1d6ce..001d218e 100644
--- a/packages/components-native/src/Divider/Divider.style.ts
+++ b/packages/components-native/src/Divider/Divider.style.ts
@@ -9,6 +9,9 @@ export const styles = StyleSheet.create({
     borderBottomWidth: JobberStyle["border-base"],
     borderBottomColor: JobberStyle["color-border"],
   },
+});
+
+export const sizeStyles = StyleSheet.create({
   large: {
     borderBottomWidth: JobberStyle["border-thick"],
     opacity: 0.875,
diff --git a/packages/components-native/src/Divider/Divider.tsx b/packages/components-native/src/Divider/Divider.tsx
index 03ebf72f..efe09d06 100644
--- a/packages/components-native/src/Divider/Divider.tsx
+++ b/packages/components-native/src/Divider/Divider.tsx
@@ -1,6 +1,6 @@
 import React from "react";
 import { View } from "react-native";
-import { styles } from "./Divider.style";
+import { sizeStyles, styles } from "./Divider.style";
 
 interface DividerProps {
   /**
@@ -8,14 +8,14 @@ interface DividerProps {
    *
    * @default "base"
    */
-  readonly size?: "base" | "large" | "largest";
+  readonly size?: "base" | keyof typeof sizeStyles;
 }
 
 export function Divider({ size = "base" }: DividerProps): JSX.Element {
   const style = [
     styles.divider,
-    size === "large" && styles.large,
-    size === "largest" && styles.largest,
+    size === "large" && sizeStyles.large,
+    size === "largest" && sizeStyles.largest,
   ];
   return <View testID="Divider" style={style} />;
 }

```

Reply:
What if we just change the `divider` property to `base`? See the new changes